recipe/_run_test.py

```diff
@@ -0,0 +1,5 @@
+from __future__ import annotations
```

Comment:
Mostly out of curiosity, why do we need annotations for this test? I ran the build without this line and it seemed to work, but I figured there's a good reason for it.

Reply:
It may not be needed. I just took part of the upstream test: https://github.com/python/typeshed/blob/main/stubs/protobuf/%40tests/test_cases/check_struct.py